GDJS/GDJS/IDE/ExporterHelper.h

```diff
@@ -48,7 +51,8 @@ struct PreviewExportOptions {
         playerId(""),
         playerUsername(""),
         playerToken(""),
-        allowAuthenticationUsingIframeForPreview(false){};
+        allowAuthenticationUsingIframeForPreview(false),
+        captureOptions({}) {};
```

Comment:
As CaptureOptions is a class, the default constructor will be called, no need to add this line.